tests/ttnn/python_api_testing/sweep_tests/ttnn_ops.py

```diff
@@ -668,11 +668,14 @@ def eltwise_add_and_apply_activation(
     output_mem_config,
     **kwargs,
 ):
+    if activation is not None:
+        activations = [activation]
+    else:
+        activations = None
```

Comment:
don't need this branch, right?

Reply:
@ayerofieiev-tt If condition checks `activation`, not `activations`. I can remove the `else` and move `activations = None` to be before the `if` but I can't get rid of it completely